extension/content.css

```diff
@@ -82,8 +82,17 @@
 }
 
 /* remove commentbox toolbar */
-.toolbar-commenting {
-	display: none !important;
+.toolbar-group {
+    display: none;
+}
+.toolbar-group:last-child {
+    display: inline-block;
+}
+.toolbar-group .toolbar-item {
+    display: none;
+}
+.toolbar-group .js-saved-reply-container {
+	display: block;
```

Comment:
@paulmolluzzo I'm actually saying that because of this comment from @sindresorhus: https://github.com/sindresorhus/refined-github/pull/22#discussion_r53374626

I'd prefer to not use `!important`, but there's a point to using it so if GitHub changes some CSS and our selector weight becomes less than GitHub selector's weight we'd need to change it. If we use `!important` changes are quite rarely needed. However I think CSS rules from extensions have a higher weight than the page CSS (unless the page CSS uses `!important`). It would be worth looking into that and removing all `!important` directives when not needed.